.github/CODEOWNERS

```diff
@@ -34,6 +34,7 @@
 /setup.cfg                              @DataDog/agent-platform
 /repository.datadog.yml                 @DataDog/agent-platform
 /generate_tools.go                      @DataDog/agent-platform
+/service.datadog.yaml                   @DataDog/agent-platform
```

Comment:
this might change soon, don't know if you can already assign to the correct one, as teams exist on github

Reply:
assigned to BARX